### PR TITLE
Minor cleanups, mostly spelling and simplification

### PR DESCRIPTION
**Pull request by rorygraves**

### CODE REVIEW

File: server/src/main/resources/reference.conf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 com.olegych.scastie.web {
   production = false
-  sesssion-secret = "WWItju7orWthk7vbAPqI72XOBCfZFxbVjMH169o9eLjHmMCGXw2VdBsQeTNF3WH0"
```
Comment (rorygraves) [inline server/src/main/resources/reference.conf]: If you have a custom deployment configuration defined somewhere else then you need to update that as well.

Comment (MasseGuillaume) [inline server/src/main/resources/reference.conf]: thank for the warning ;-)

Comment (MasseGuillaume) [inline server/src/main/resources/reference.conf]: wow this was actually a security vulnerability. In the production config it's spelled correctly. It was using the development secret in production.

**Comment by MasseGuillaume** [comment]
Thanks!

**Pull request merged**